src/watchmaker/utils/__init__.py

```diff
@@ -119,3 +119,20 @@ def clean_none(value):
         return None
 
     return value
+
+
+def copy_subdirectories(src_dir, dest_dir, log):
```

Comment:
make `log` optional, otherwise the `if log` test doesn't make much sense
```suggestion
def copy_subdirectories(src_dir, dest_dir, log=None):
```